pina/condition/condition.py

```diff
@@ -85,7 +85,20 @@ class Condition:
     )
 
     def __new__(cls, *args, **kwargs):
-
+        """
+        Create a new condition object based on the keyword arguments passed.
+
+        - ``input`` and ``target``: :class:`InputTargetCondition`
```

Comment:
this will not work, use full path :class:"~pina.condition.input_target_condition. InputTargetCondition" (similarly below), use ` instead of "